backend/apps/feature_requests/signals.py

```
@@ -0,0 +1,92 @@
+                "data": {
+                    "id": str(feature.id),
+                    "title": feature.title,
+                    "upvotes": feature.upvotes,
+                    "downvotes": feature.downvotes,
+                    "total_votes": feature.total_votes,
+                    "weighted_votes": feature.weighted_votes,
+                    "priority_score": feature.priority_score,
+                }
+            }
+        )
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)